### PR TITLE
Make selection behaviour match ember-frost-list

### PR DESCRIPTION
**Pull request by AdamWard1995**
### This project uses [semver](semver.org), please check the scope of this pr:

- [ ] #none# - documentation fixes and/or test additions
- [X] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

This PR prevents text from being selected when doing a range select and when clicking on a row body (as opposed to the checkbox) that row will be the only selected one

# CHANGELOG
 * **Fixed** selection functionality to match `ember-frost-list`

### CODE REVIEW

**Comment by job13er** [comment]
I don't use table selection, and am not familiar with the implementation/behavior in `frost-list`, so I'm probably not the best reviewer for this. Looks like @quincyle is the biggest contributor on `ember-frost-list`, so maybe he can take a peek.

File: addon/components/frost-fixed-table.js [hunk 1]
```diff
@@ -421,10 +423,26 @@ export default Component.extend({
...
   // == Lifecycle Hooks =======================================================
 
+  init () {
+    this._super(...arguments)
+    this._keyHandler = this.setShift.bind(this)
+    $(document).on(`keyup.${this.elementId} keydown.${this.elementId}`, this._keyHandler)
+  },
+
+  willDestroy () {
+    $(document).off(`keyup.${this.elementId} keydown.${this.elementId}`, this._keyHandler)
```
Comment (dafortin) [inline addon/components/frost-fixed-table.js]: Should we release the bind on the destroy?

Comment (dafortin) [inline addon/components/frost-fixed-table.js]: call this._super?

Comment (AdamWard1995) [inline addon/components/frost-fixed-table.js]: Definitiely missing call to _super, but not sure what you mean by if we should release the binding on destroy. This is what we do in ember-frost-list https://github.com/ciena-frost/ember-frost-list/blob/master/addon/components/frost-list.js#L162 and it seems like the logical hook to do this https://guides.emberjs.com/v2.14.0/components/the-component-lifecycle/#toc_detaching-and-tearing-down-component-elements-with-code-willdestroyelement-code

**Comment by quincyle** [comment]
Approved

**Pull request merged**